.npmignore

```diff
@@ -15,10 +15,3 @@ demo
 demo-build
 webpack.config.js
 
-#typescript sources
```

Comment:
ok, can do i needed it for using it in my project, as i could not get the fixed version via npm!